pandas/core/series.py

```diff
@@ -1421,7 +1421,7 @@ def to_string(
     def to_markdown(
         self,
         buf: Optional[IO[str]] = None,
-        mode: Optional[str] = None,
+        mode: str = "wt",
```

Comment:
you are setting for all text based backend?

Reply:
would need to update the doc-strings i think for that as well

Reply:
The markdown writer produces text, but you can still pass whatever mode here.
Will update the doc

Reply:
Does `to_markdown` not get `storage_options`?